### DISCUSSION TITLE
Scheduled monitoring

### DISCUSSION THREAD

**Discussion by tsvet551** [Nachtzuster/BirdNET-Pi#254] [Ideas]
Greetings! Would it be great if the BirdNET-Pi had the ability to activate audio monitoring at certain times of the day? For example, to monitor night and twilight birds (owls, nightjars, etc.), so as not to monitor all birds 24/7 and save battery power in an autonomous installation. Or is there such a function, but I don't know about it? Thanks!

**Comment by drunkenungulate** [comment]
At least for a Pi4 there probably isn't so much battery power to be saved by disabling the recording and analysis. Mine runs right around 6% processor usage, and I wouldn't expect much in the way of incremental peripheral power usage from the BirdNET-Pi related processes. If it was pulling a lot higher CPU usage then maybe this would help. 

If you want to save power there are options though. This isn't specific to BirdNET-Pi, but I'm using a WittyPi 4 L3V7 board on mine. It has a charge controller for 3.7v Lithium ion/polymer batteries and a realtime clock. You can use it to power a Raspberry Pi, seamlessly go from external to battery power, and schedule startup/shutdown, including a graceful shutdown when the battery is low. 

If you already have the battery power sorted then their simpler WittyPi 4 model would probably be enough. It has the RTC and power scheduling functions but no battery interface.

**Comment by tsvet551** [comment]
Thanks for the advice, colleague! Indeed, the application itself consumes little energy, but I plan to completely disable the RaspPi and then give the command to boot the operating system again. This operation will save energy, I hope  :-))   I turned off HDMI and Wi-Fi. Maybe the "cron" team will help me, it is in the BirdNET-pi assembly?

**Reply by alexbelgium** [reply]
Hi, this should be rather easy. I'm not a great cron expert but this should do the job : 

Open crontab with the command
`crontab -e`

Add the following lines :
```
# Restart services at 8pm
0 20 * * * services=(chart_viewer.service spectrogram_viewer.service icecast2.service birdnet_recording.service birdnet_analysis.service birdnet_log.service birdnet_stats.service); for i in "${services[@]}"; do sudo systemctl restart "${i}"; done; exit 0
# Stop services at 8am
0 8 * * * services=(chart_viewer.service spectrogram_viewer.service icecast2.service birdnet_recording.service birdnet_analysis.service birdnet_log.service birdnet_stats.service); for i in "${services[@]}"; do sudo systemctl stop "${i}"; done; exit 0
```

Save and exit. To see running cron jobs your should see them with 
 `crontab -l`

**Reply by unosonic** [reply]
if you want to completely switch off the pi, cron won't help you. for that task I can recommend [wittpiy4](https://www.uugear.com/product/witty-pi-4/). It allows complex ON/OFF schedule plans plus some extra features, power consumption in OFF state is only a few micro amps.

**Reply by tsvet551** [reply]
> if you want to completely switch off the pi, cron won't help you. for that task I can recommend [wittpiy4](https://www.uugear.com/product/witty-pi-4/). It allows complex ON/OFF schedule plans plus some extra features, power consumption in OFF state is only a few micro amps.

Thanks!

**Comment by tsvet551** [comment]
Thanks! I will definitely try it! In the BirdNet-Pi assembly from Nachtzuster, the crown is included in the software?

**Reply by tsvet551** [reply]
It seems there is, "crontab -e" worked, I will try, thank you.
